examples/advanced/tailwind-preset/config/format.js

```diff
@@ -0,0 +1,56 @@
+import { isColor } from './filter.js';
+
+/**
+ * Exports tailwind plugin for declaring root CSS vars
+ * @see https://tailwindcss.com/docs/plugins#overview
+ */
+export const cssVarsPlugin = ({ dictionary, options }) => {
+  const vars = dictionary.allTokens
+    .map((token) => {
+      const value = options.usesDtcg ? token.$value : token.value;
+      return `'--${token.name}': '${value}'`;
+    })
+    .join(',\n      ');
+
+  return `import plugin from 'tailwindcss/plugin';
+
+export default plugin(function ({ addBase }) {
+  addBase({
+    ':root': {
+      ${vars},
+    },
+  });
+});\n`;
+};
```

Comment:
I'll definitely try that out. I wasn't aware of the [accessibility issues](https://github.com/prettier/prettier/issues/7475#issuecomment-668544890)- thanks for sharing.

Reply:
I like this a lot... it feels cleaner than hardcoding spaces. VSCode users can add `"editor.tabSize": 2` to their `settings.json` or install [this plugin](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) to pick up the `.editorconfig` settings

Reply:
It seems the tab width setting isn't applied to code blocks in markdown files... I'll update when I find a solution there. 

https://github.com/orgs/community/discussions/8067

Reply:
Keeping spaces [for markdown files](https://github.com/jessicalynch/style-dictionary/blob/add-tailwind-example-v4/.editorconfig#L17) could work for now.

Reply:
https://github.com/orgs/community/discussions/77604 seems like they haven't yet followed up on the person who raised this issue.

For now I think it's fine to ignore the fact that GitHub does not render markdown code blocks properly, just doesn't respect the tab width preference from the GH profile settings -> appearance. It's unfortunate, but that's just a GH issue, I don't think it's the right choice to work around it by keeping spaces for MD files.

As for editorconfig, we haven't updated it in 4 years and I'm kinda down to just delete it since we use Prettier for formatting our code. I don't think there's a use case for having it anymore.

Reply:
Yeah it's too bad... but yes, I'll remove that MD override. I see there is prettier config in the package.json- didn't notice that. I can remove the .editorconfig file and migrate the tailwind example override to the package.json config _(pending the separate issue to re-format everything with the new settings so this diff isn't crazy)_.

Alternatively, the package.json config could be removed in favor of the .editorconfig file, since [prettier will recognize it](https://prettier.io/docs/en/configuration.html#editorconfig). I like the idea of a standardized config, but I'm not sure how much adoption it has. I found a couple issues raised in the Ruff and Black repos, and it seems they're not planning to support it for now, unfortunately.

Reply:
Deleted the unused .editorconfig file 👌